### PR TITLE
PLT-2644 Refactor login, claim and create_team into views and add actions

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: webapp/utils/web_client.jsx [hunk 1]
```diff
@@ -21,18 +21,6 @@ class WebClientClass extends Client {
         this.setTeamId(TeamStore.getCurrentId());
     }
 
-    track = (category, action, label, property, value) => {
```
Comment (coreyhulen) [inline webapp/utils/web_client.jsx]: How does this work with stuff like https://github.com/mattermost/mattermost-driver-javascript/blob/master/client.jsx#L175 ?

Comment (jwilander) [inline webapp/utils/web_client.jsx]: Right, my grep missed that one

**Comment by crspeller** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Pull request merged**